index.js

```diff
@@ -323,14 +354,30 @@ class Store {
     utils.tryUnlink(this.path);
   }
 
+  /**
+   * Read and parse the data from the file system store. This method should
+   * probably not be called directly. Unless you are familiar with the inner
+   * workings of the code it's recommended that you use .load() instead.
+   *
+   * ```js
+   * data = store.readPraseFile();
+   * ```
+   * @name .readParseFile
+   * @return {undefined}
+   */
+
+  readParseFile() {
+    return JSON.parsefs.readFileSync(this.path);
```

Comment:
There's a typo here... it looks like `.load` is never tested so this wasn't caught in the unit tests.
```suggestion
    return JSON.parse(fs.readFileSync(this.path));
```

Reply:
Doh!